raiwidgets/raiwidgets/dashboard.py

```diff
@@ -106,7 +108,7 @@ def load_widget_file(self, path):
             content = content.replace(
                 "__rai_model_data__",
                 json.dumps(self.model_data,
-                           default=_safe_object_serializer))
+                           default=serialize_json_safe))
```

Comment:
this doesn't look right, it doesn't set forObject=True

Reply:
Discussed offline new design for serialization